### DISCUSSION TITLE
integration with django

### DISCUSSION THREAD

**Discussion by SadeghKrmi** [facebook/docusaurus#8411] [Q&A]
Hi,

I am wondering how is possible to integrate **Docusaurus** with **Django** as backend? What is the best method to do so?

**Comment by slorber** [comment]
Docusaurus builds a static site. 

Django should be able to serve the static files that Docusaurus buids, in the same way it can serve any file like a pdf or an excel file.

https://docs.djangoproject.com/en/4.1/howto/static-files/deployment/

That's outside the scope of Docusaurus, which is only responsible for emitting HTML, CSS and JS files, and doesn't have strong opinions on how you should serve those files.

**Reply by SadeghKrmi** [reply]
There is a challenge, a big one, which is in Django there is a need to tag js/css with specific static tag, but after Docusaurus build, it's not possible to go through all the js/css to tag them, it will be a big challenge, there are many of them.
Any idea?

**Reply by slorber** [reply]
no idea. it looks more like a limitation of Django and you should rather discuss that with the Django team directly
